### PR TITLE
section header removed

### PR DESCRIPTION
**Pull request by Adityachaturvedi2906**

### CODE REVIEW

File: src/afch.js [hunk 1]
```diff
@@ -27,6 +27,7 @@
 
 	// Set up constants
 	AFCH.consts = {};
+	AFCH.consts.mockItUp = true;
```
Comment (enterprisey) [inline src/afch.js:30]: Please remove this line from the change, as it was only needed for testing

**Review by enterprisey — CHANGES_REQUESTED** [review]
Looks good - please make the indicated change.

**Comment by enterprisey** [comment]
Thanks for your work on this!

**Comment by Adityachaturvedi2906** [comment]
> Thanks for your work on this!

Its was my pleasure ☺

**Pull request merged**